src/evaluate.py

```diff
@@ -153,6 +165,7 @@ def write_preds(
             + ["wmt"]
             + tasks_module.ALL_COLA_NPI_TASKS
         )
+        simple_csv_preds_tasks = ["qasrl"]
```

Comment:
Adding it as a static attribute of tasks may not be a good idea: we would either have to define that attribute for all tasks (including tasks which already have specific prediction-writing logic defined), or have to use a `hasattr` which is quite ugly. 

Since `evaluate.py` has checks against task classes, I could alternatively add
```
SIMPLE_PRED_CSV_TASKS = [QASRLTask]
```
as a constant to the file.

Reply:
IIRC, all tasks inherit from the same class. How about we add csv_preds = False to that class, and override it here? I think Yada does something similar in another PR for another task attribute.